backend/api/utils.py

```diff
@@ -42,4 +44,9 @@ def get_sort_order(query_params: QueryDict, collection: Collection):
 
 
 def get_new_id() -> str:
```

Comment:
How do you ensure that the IDs will be unique?

Reply:
Why are you using both `nanoid` and `uuid` ? We can use `uuid` only right ? can you explain ?

Reply:
Oh, it was just to show them two. I'll remove the one we don't keep.

Reply:
As it is, I'm not absolutely sure it is unique. But the colliding rate is said to be quite low...
What I could do is to generate a new ID until the generated one is not already existing.

Reply:
Hum ok i see. Is there a way to use the timestamp as well in the id ? Or while generating ? It might ensure unicity.